packages/db/src/schema/target-group.ts

```diff
@@ -13,6 +13,7 @@ export const targetMetadataGroup = pgTable("target_metadata_group", {
   name: text("name").notNull(),
   description: text("description").notNull(),
   keys: text("keys").array().notNull(),
+  kinds: text("kinds").array(),
```

Comment:
I kept this optional because it is. If we want a different status quo we can do that.